src/components/FilterList.jsx

```diff
@@ -154,13 +154,17 @@ const FilterList = ({
   /*************************/
 
   const [apiEndpoint, setApiEndpoint] = useState(
-    () => (defaultApiEndpoint = buildDefaultEndPoint())
+    () => (defaultApiEndpoint = buildDefaultEndPoint(includeDateFilter))
   );
 
   // This is a hook triggered when the querystring in the URL is changed.
   useEffect(() => {
     setFilters((filters) => UpdateFilters(filters, location.search));
-    setApiEndpoint(buildDefaultEndPoint(isDateClear ? false : true));
+    setApiEndpoint(
+      includeDateFilter
+        ? buildDefaultEndPoint(isDateClear ? false : true)
```

Comment:
The isDateClear was still passing a true here even when the date wasnt to be displayed. So first we check if dates exist then we check if they have been cleared.